cppcache/include/geode/Cache.hpp

```diff
@@ -239,6 +241,10 @@ class APACHE_GEODE_EXPORT Cache : public GeodeCache {
 
   SystemProperties& getSystemProperties() const override;
 
+  void setLogLevel(LogLevel newLevel);
```

Comment:
Please add documentation to these new methods.

Reply:
done